notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,741 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91f937b4-5542-4445-922c-d544547d3d23",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### SlewDataSet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a7f87c6b-60b4-4099-88d0-a8cf961bfdd2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "@dataclass\n",
+    "class SlewDataSet:\n",
+    "    \"\"\"\n",
+    "    Hold all the data associated with a slew.\n",
+    "    \n",
+    "    Attributes\n",
+    "    ----------\n",
+    "    name : str\n",
+    "        Human readable name\n",
+    "    begin : `astropy.time.Time`\n",
+    "        Start date and time in ISO format and in UTC.\n",
+    "    end : `astropy.time.Time`\n",
+    "        End date and time in ISO format and in UTC.\n",
+    "    df : pd.DataFrame | None = None\n",
+    "        Table containing the HP measured forces (the columns should be \n",
+    "        `measuredForces0`, `measuredForces1`, etc.). The index should be \n",
+    "        a timestamp. \n",
+    "    stats : pd.DataFrame | None = None\n",
+    "        Table containing the min/max/ptp values for each HP measured forces \n",
+    "        in the beginning of a slew and at the end of a slew. \n",
+    "    events : list[TMAEvent] | None = None\n",
+    "        TMA slew events obtained via TMAEventMaker to have a more exact \n",
+    "        timestamp from MTMount.\n",
+    "    \"\"\"\n",
+    "    name : str\n",
+    "    begin : Time\n",
+    "    end : Time\n",
+    "    df : pd.DataFrame | None = None\n",
+    "    stats : pd.DataFrame | None = None\n",
+    "    events : None = None"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9db158a6-16f2-484d-b7a5-54228068fe38",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### query_dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7888dcd-8ded-4c7a-8aec-e35ffd2cf66d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "async def query_dataset(start, end):\n",
+    "    \"\"\"\n",
+    "    Queries all the relevant data, resample them to have the same requency and merge them in a single dataframe.\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    start : str\n",
+    "        Start date and time in ISO format and in UTC.\n",
+    "    end : str \n",
+    "        End date and time in ISO format and in UTC.\n",
+    "        \n",
+    "    Returns\n",
+    "    -------\n",
+    "    pd.DataFrame\n",
+    "    \"\"\"    \n",
+    "    hp_measured_forces = await efd_client.select_time_series(\n",
+    "        \"lsst.sal.MTM1M3.hardpointActuatorData\", \n",
+    "        measured_forces_topics, \n",
+    "        Time(start, scale='utc'), \n",
+    "        Time(end, scale='utc')\n",
+    "    )\n",
+    "    hp_measured_forces = hp_measured_forces.resample(\"100ms\").mean()\n",
```

Comment:
Why is there mean resample? That just smears max/min in statistics. There isn't a need to resample the data.

Reply:
Removed the resample as suggested.